Maddori.Apple/Maddori.Apple/Screen/InProgress/InProgressViewController.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  InProgressViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/24.
+//
+
+import UIKit
+
+import SnapKit
+
+final class InProgressViewController: BaseViewController {
+    
+    private var keywordData = Keyword.mockData
+    private let currentRetrospectiveUser = "진저"
+//    private let user = "이드"
+    private let user = "진저"
+    private enum Size {
+        static let keywordLabelHeight: CGFloat = 50
+        static let sectionPadding: CGFloat = 60
+    }
+    private var keywordsSectionList = [[Keyword]]()
+    private var isUserRetrospective: Bool {
+        return user == currentRetrospectiveUser ? true : false
+    }
+    
+    // MARK: - properties
```

Comment:
p5;

이거도 진짜 이상한 사람같이 보일 수 있는데.... 
```suggestion
    // MARK: - property
```
는 어떨까요 ???
함수 부분 주석도 funcs라 쓰지않고 func라 쓰고있고, extension 부분도... 하하하하하ㅏㅏㅏ하 
p5 입니다 p5 ! 
<img width="440" alt="image" src="https://user-images.githubusercontent.com/78677571/199131707-15f2f190-5a4d-4ba5-8c1c-ac9c02d1ab3a.png">

Reply:
[d8f60e7](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/59/commits/d8f60e741b979369372b7b6cd27ef1f10d5b9443)
헿ㅎ 감사합니닿ㅎㅎ